libopenage/util/file.h

```diff
@@ -38,41 +42,68 @@ template<typename lineformat>
 void read_csv_file(const std::string &fname, std::vector<lineformat> &out) {
```

Comment:
i'd add a new `read_csv_file` which does not open the file but instead uses the data from the map. we have to do this when deglobalizing the map.

alternatively, we can add another optional argument which provides the file data already (from the map), so the open and read is bypassed and the map data is used instead.